setup.py

```diff
@@ -7,9 +7,11 @@
 
 from setuptools import setup
 
-if not sys.version_info[:2] >= (3, 6):
-    sys.exit("conda is only meant for Python 3.6 and up.  "
-             "current version: %d.%d" % sys.version_info[:2])
+if not sys.version_info[:2] >= (3, 7):
+    sys.exit(
+        f"conda is only meant for Python 3.7 and up. "
+        f"current version: {sys.version_info.major}.{sys.version_info.minor}"
+    )
```

Comment:
@kenodegard Technically the `python_requires` line is only evaluated during building and installation but setuptools/distutils provides the ability to specify additional subcommands that may not do that. TBH if we were to ever release conda again to PyPI we should just migrate over to using pyproject.toml etc instead of keeping setup.py around.